src/components/contentItem/ContentItem.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
с 17 версии писать React не нужно
```suggestion
import { useEffect, useState } from 'react';
```